crates/re_types/definitions/rerun/archetypes/disconnected_space.fbs

```diff
@@ -5,13 +5,11 @@ include "rerun/components.fbs";
 
 namespace rerun.archetypes;
 
-/// Specifies that the entity path at which this is logged is disconnected from its parent.
+/// Specifies that the entity path at which this is logged is spatially disconnected from its parent, making it impossible to transform the entity path into its parent's space and vice versa.
```

Comment:
That is a very long line :scream:

Reply:
The first sentence/line/paragraph in a docstring should be short, as it shows up in the summary view of the docs:

<img width="1016" alt="Screenshot 2024-01-29 at 23 04 33" src="https://github.com/rerun-io/rerun/assets/1148717/4c67f435-9eab-4155-872a-936ee2e9f7b0">

Consider it a challenge to make it short and useful!

Maybe `/// Spatially disconnect this entity from its parent.` ?

Reply:
ok sounds good